package.json

```diff
@@ -27,23 +28,30 @@
     "@tsndr/cloudflare-worker-jwt": "^1.1.4",
     "add": "^2.0.6",
     "classnames": "^2.3.1",
-    "next": "^12.1.6",
+    "next": "^12.3.0",
     "next-seo": "^4.24.0",
-    "react": "^18.1.0",
-    "react-dom": "^18.1.0",
-    "react-icons": "^4.2.0"
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.2.0",
+    "uuid": "^8.3.2"
```

Comment:
not sure that we actually need this dependency as there's a UUID generator built into node using the `crypto` module.  so uninstall this guy.